R/dataset.R

```diff
@@ -88,6 +88,42 @@ setMethod("notes<-", "CrunchDataset", function (x, value) {
     invisible(setEntitySlot(x, "notes", value))
 })
 
+
+#' Name, alias, and description for Crunch objects
```

Comment:
Give this man page a title that matches this function, and it probably merits an additional sentence or two that explains what a `pk` is and what effect it has.